ckanext/validation/jobs.py

```diff
@@ -108,9 +117,16 @@ def run_validation_job(resource):
 
 def _validate_table(source, _format=u'csv', schema=None, **options):
 
-    report = validate(source, format=_format, schema=schema, **options)
+    http_session = options.pop('http_session', None) or requests.Session()
+
+    use_proxy = 'ckan.download_proxy' in t.config
+    if use_proxy:
+        proxy = t.config.get('ckan.download_proxy')
+        log.debug(u'Download resource for validation via proxy: %s', proxy)
+        http_session.proxies.update({'http': proxy, 'https': proxy})
+    report = validate(source, format=_format, schema=schema, http_session=http_session, **options)
 
-    log.debug(u'Validating source: {}'.format(source))
+    log.debug(u'Validating source: %s', source)
```

Comment:
was this from merging upstream? it doesn't look related

Reply:
yeah, from upstream https://github.com/open-data/ckanext-validation/commit/37ef0bac12c924248cb7ea523fa56b47cfe76a97